### PR TITLE
Continue: Update CONTRIBUTING.md with pictures and instructions

### PR DESCRIPTION
**Pull request by Michelle-Lohwt**
Please review the changes made:
- add/ modify images per request
- move images to assets/images
- add bash to codes

Note: The previous pull request was `forced push` so cannot reopen the PR

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -3,24 +3,122 @@
...
 
+---
+
 ### Step 1: Fork the project
+* Fork this Repository. This will create a Local Copy of this Repository on your Github Profile.
+![fork](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic1_fork.png)
+
+* Go to `Code` -> `HTTPS` to copy the link: <br>
+![fork2](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic9_fork.png)
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:29]: Add new image per request

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -3,24 +3,122 @@
...
+```bash
+$ git clone https://github.com/<your-username>/Coden.github.io 
+    
+# This is a comment, change **repo_name** in the following to your repository name
+$ cd repo_name
+    
+$ git remote add upstream https://github.com/CodenCommunity/Coden.github.io
+```
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:40]: Add `bash` to code

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -3,24 +3,122 @@
...
 * Click [here](https://github.com/CodenCommunity/Support/issues/new?assignees=LeeRenJie&labels=Join+Coden+GitHub&template=join-coden-github.md&title=Join+Coden+on+GitHub) to be a member of Coden on GitHub to contribute to any in-house projects.
 * Click [here](https://discord.gg/rSKKKZEnVn) to join us in our Discord Server and meet the rest of the community!
 Once you are on GitHub and Discord, move on to step 1
 
+---
+
 ### Step 1: Fork the project
+* Fork this Repository. This will create a Local Copy of this Repository on your Github Profile.
+![fork](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic1_fork.png)
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:26]: fix link:
from: `assets/pic1_fork.png` to `assets/images/pic1_fork.png`

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -3,24 +3,122 @@
...
+```
+
+---
+
 ### Step 2: Find an issue
+* Take a look at the Existing Issues or create your **own** Issues!  
+* Wait for the Issue to be assigned to you after which you can start working on it.  
+* Note : Every change in this project should/must have an associated issue.
+![issue](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic2_find_issue.png)
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:48]: fix link

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -3,24 +3,122 @@
...
+---
+  
 ### Step 3: Create a branch
+* Create a new branch in your computer directory. Use its name to identify the issue you are addressing.
+  
+```bash
+# Change **branch_name** in the following to the name of issue you are addressing
+$ git checkout -b branch_name
+```
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:58]: Add `bash`

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -3,24 +3,122 @@
...
+* OR you can create the branch in Github as shown here: 
+![branch](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic3_branch.png)
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:61]: fix link

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -3,24 +3,122 @@
...
+* OR you can create the branch in Github as shown here: 
+![branch](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic3_branch.png)
+    
+```bash
+# After you create the branch here, remember to update it in your computer directory
+$ git pull
+$ git checkout branch_name
+```
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:67]: Add `bash`

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -3,24 +3,122 @@
...
+---
+  
 ### Step 4: Work on the issue assigned
+* Work on the issue(s) assigned to you.   
+* Add all the files/folders needed.  
+* After you've made changes or made your contribution to the project add changes to the branch you've just created by using terminal:  
+```bash
+# To add all new files to branch Branch_Name  
+$ git add .  
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:77]: Add `bash`

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -3,24 +3,122 @@
...
+# To add only a few files to Branch_Name
+$ git add <some files>
+```
+- Using GitHub
+![upload_file](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic4_upload_file.png)
+
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:83]: fix link

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -3,24 +3,122 @@
...
+---
+
 ### Step 5: Commit
+* You are suggested to use Conventional Commits in your commit message to ease other contributors reading the documentation, read [here](https://www.conventionalcommits.org/en/v1.0.0/)
+* To commit give a descriptive message for the convenience of reviewer by using terminal:  
+```bash
+# This message get associated with all files you have changed  
+$ git commit -m "message"  
+```
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:92]: Add `bash`

File: .github/CONTRIBUTING.md [hunk 11]
```diff
@@ -3,24 +3,122 @@
...
+- Using GitHub
+![commit](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic5_commit.png)
+---
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:95]: fix link

File: .github/CONTRIBUTING.md [hunk 12]
```diff
@@ -3,24 +3,122 @@
...
+  
 ### Step 6: Push it to your forked branch
+* Push your changes to the forked branch
+```bash
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:99]: Add `bash`

File: .github/CONTRIBUTING.md [hunk 13]
```diff
@@ -3,24 +3,122 @@
...
+git push
+```
+* You should see something like this in your Github after pushing:
+![push](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic6_push.png)
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:103]: fix link

File: .github/CONTRIBUTING.md [hunk 14]
```diff
@@ -3,24 +3,122 @@
...
+---
+  
 ### Step 7: Make a Pull Request (PR)
+* Go to your repository in browser and click on compare and pull requests. Then add a title and description to your pull request that explains your contribution.
+![pull_request](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic7_create_pull.png)
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:108]: - Modify pic7 as requested (point arrow to `compare and pull`)
- fix link

File: .github/CONTRIBUTING.md [hunk 15]
```diff
@@ -3,24 +3,122 @@
...
+* AND compare the forked branch with the main branch
+![compare](https://github.com/Michelle-Lohwt/Coden.github.io/blob/branch1/assets/images/pic8_compare_repo.png)
```
Comment (Michelle-Lohwt) [inline .github/CONTRIBUTING.md:110]: fix link

**Comment by LeeRenJie** [comment]
lgtm

**Pull request merged**